includes/features/SU-SWS/stanford_jumpstart_home/homepage-mayfield-dynamic.feature

```diff
@@ -3,13 +3,13 @@ Feature: Mayfield Homepage Layout
   As an end user
   I want to check for the existence of content that should appear
 
-  @api @dev @destructive @jsa
+  @api @dev @destructive
   Scenario: Set the homepage to Mayfield news and events
     Given I am logged in as a user with the "administrator" role
     And I am on "admin/stanford-jumpstart/customize-design"
     Then I press the "edit-layouts-stanford-jumpstart-home-mayfield-news-events-selector" button
 
-  @deploy @jsa @live @safe
+  @safe
   Scenario Outline: Header content for mayfield with news and events.
```

Comment:
Hmmm.... @kmakitan, I'm wondering if we should come up with a better strategy for testing home page layouts. Since we don't want to enable and test the various home page layouts on a live site, these tests should be either @dev or @deploy. Your thoughts?